### ISSUE TITLE
Data loading consistency in training and testing

### ISSUE THREAD

**Issue by remindof** [Cyril-Sterling/EVFlowNet-pytorch#4]
Dear author,

Thank you for your hard work on this project!

In _data_loader.py_, I found there is different output values in **event_time_image** when I set 
data = EventData('xxxx/mvsec/', '**train**') and
data = EventData('xxxx/mvsec/', '**test**').

Particularly, in training mode, if you run _data_loader.py_ alone, there will be output values around "448" for variable _a_. However, in testing mode, the values are around "1.5" for the same variable.

I checked the code and found that here:
https://github.com/Cyril-Sterling/EVFlowNet-pytorch/blob/fdd800d0fc546cf63c516c59b478be04ed3b5d4b/src/data_loader.py#L105
You multiply event time image by 255; However in testing mode, you directly concatenate event count image and event time image together.

When I run your original code, the network almost learns nothing... I wonder if the above issue caused the problem?

Thank you!

**Comment by sankeerth95** [comment]
Yes it does seem to be a mistake, the 255 is supposed to be multiplied to event_count_image instead. However even with that change in place, it did not train.  I was wondering if there's any version which did manage to converge?
